lib/Money/Money.php

```diff
@@ -297,4 +297,17 @@ public static function stringToUnits( $string )
 
         return (int) $units;
     }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @return array
+     */
+    function jsonSerialize()
```

Comment:
also use visibility here, like `public function jsonSerialize()`  to follow the general coding style and PSR standard